docs/resources/alert_rule.md

```diff
@@ -28,7 +28,7 @@ resource "valtix_alert_rule" "rule2" {
   type          = "Type_SystemLogs"
   sub_type      = "SubType_SystemLogsAccount"
   severity      = "Info"
-  is_active     = true
+  is_active     = "true"
```

Comment:
would this work? I thought boolean values must always be without quotes. can you please confirm